api/client/connection.go

```diff
@@ -267,74 +253,49 @@ func (c *Connection) SearchRaw(ctx context.Context, search api.SearchRequest, pa
 //		fmt.Println(rec)
 //	}
 //
-func (c *Connection) Search(ctx context.Context, spaceID api.SpaceID, query string) (*ZngSearch, error) {
+func (c *Connection) Search(ctx context.Context, id ksuid.KSUID, query string) (*ZngSearch, error) {
 	procBytes, err := c.ZtoAST(ctx, query)
 	if err != nil {
 		return nil, err
 	}
 	r, err := c.SearchRaw(ctx, api.SearchRequest{
-		Space: spaceID,
-		Proc:  procBytes,
-		Dir:   -1,
+		Pool: id,
+		Proc: procBytes,
+		Dir:  -1,
 	}, nil)
 	if err != nil {
 		return nil, err
 	}
 	return NewZngSearch(r), nil
 }
 
-func (c *Connection) IndexSearch(ctx context.Context, space api.SpaceID, search api.IndexSearchRequest, params map[string]string) (*ZngSearch, error) {
+func (c *Connection) IndexSearch(ctx context.Context, id ksuid.KSUID, search api.IndexSearchRequest, params map[string]string) (*ZngSearch, error) {
```

Comment:
I think this wants to be an index scan (that optionally uses search indexes) like `ls -partition` (BTW, I'm not sure we should call this `ls` but that is where the equivalent zar introspection lived).  Not urgent to implement this now.